### PR TITLE
Fix the duplicate hash links in the ToC

### PR DESCRIPTION
**Pull request by csandman**
There are multiple duplicate #hash links in the table of contents that when clicked, all end up pointing to the same section of the guide.  Github adds a -1, -2 etc to any duplicate headers' ids so I added them to the Table of contents to make them functional.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,25 +25,25 @@ All automated.
     - [Setup a VPN Container](#setup-a-vpn-container)
       - [Introduction](#introduction)
       - [privateinternetaccess.com custom setup](#privateinternetaccesscom-custom-setup)
-      - [Docker container](#docker-container)
+      - [Docker container](#docker-container-1)
```
Comment (sebgl) [inline README.md:28]: These could be easier to maintain with more explicit names. For example: "#docker-container-vpn", "#docker-container-radarr", etc.

Comment (csandman) [reply README.md:28]: I thought the same thing originally, but I felt that I should submit a fix without changing the content.  I could update it so they have unique names if you'd like!

Side note, when I wrote the section on Bazarr, I did it your way specifically for that reason haha

**Review by sebgl — APPROVED** [review]
Super late: thank you!

**Comment by csandman** [comment]
No problem!  I forgot all about this haha.  I don't even really use this setup anymore (I switched to an Unraid server) but this guide was extremely helpful to me when I first got my server set up so I really appreciate it!

**Pull request merged**